packages/axios-extension/src/abap/message.ts

```diff
@@ -85,7 +85,7 @@ export function prettyPrintMessage({
 }): void {
     try {
         const jsonMsg = JSON.parse(msg) as SuccessMessage;
-        log.info(jsonMsg.message);
+        log.debug(jsonMsg.message);
```

Comment:
This is the message from the server to show to the user? Should still be info then.

Reply:
End users do not use this module directly and so all messages should be returned for display in the appropriate way depending on the UI environment. Im curious how an end user would ever see these messages from YUI or VSIX?